### ISSUE TITLE
vendoring support

### ISSUE THREAD

**Issue by slidemonk** [alecthomas/gometalinter#175]
I'm using go 1.7.1 and have some packages in vendor/ directory. However, flycheck keeps complaining about missing packages which are there in the vendor dir. Go compiler builds the project just fine, so packages are correctly placed in vendor/.

Here is output in messages buffer:
```
could not import google.golang.org/grpc (cannot find package "google.golang.org/grpc" in any of: (interfacer)

unused global variable could not import google.golang.org/grpc (cannot find package "google.golang.org/grpc" in any of: (varcheck)

could not import google.golang.org/grpc (cannot find package "google.golang.org/grpc" in any of: (unconvert)

unused struct field could not import google.golang.org/grpc (cannot find package "google.golang.org/grpc" in any of: (structcheck)

error return value not checked (could not import google.golang.org/grpc (cannot find package "google.golang.org/grpc" in any of:) (errcheck)

could not import google.golang.org/grpc (cannot find package "google.golang.org/grpc" in any of: (aligncheck)
```

**Comment by GarethLewin** [comment]
I believe you need to do `go build -i ./...` or `go install -i ./...`

Many linters use the pkg files as opposed to parsing the raw go files.

**Comment by slidemonk** [comment]
Runnig `gometalinter .` from cmdline does not give these errors. So the problem must be with emacs flycheck-gometalinter package. Closing the issue here.
